ext/BijectorsReverseDiffExt.jl

```diff
@@ -222,48 +222,33 @@ end
 @grad_from_chainrules _link_chol_lkj_from_upper(x::TrackedMatrix)
 @grad_from_chainrules _link_chol_lkj_from_lower(x::TrackedMatrix)
 
-cholesky_lower(X::TrackedMatrix) = track(cholesky_lower, X)
```

Comment:
You have to keep this line in this file, to ensure that `TrackedMatrix` does not hit the `AbstractMatrix` fallback and that the error hint works as intended.